gpkit/fmincon/simpleflight.py

```diff
@@ -0,0 +1,62 @@
+"Minimizes airplane drag for a simple drag and structure model."
```

Comment:
Sorry if I'm being really dense about this but I'm still not sure I understand how that helps. As far as I can tell `generate_example_tests` only works with the `class` structure in `t_example` and can't be used to import the model. Furthermore, `docs/source/examples/`simpleflight.py` has both a single model and a sweep model, which is more than I need. I don't really know how to include this `simpleflight` test without duplicating code. I think that there should therefore be duplicated code or this function shouldn't have a test method. Either would be fine with me, though I would prefer the duplicated code.